docs/release.md

```diff
@@ -0,0 +1,105 @@
+## Release process
+
+To release a new version, you need to be a member of `gcp-variant-transforms`,
+and `gcp-variant-transforms-test` projects and have write access to the GitHub
+repo.
+
+The process is as follows:
+
+1. Navigate to
+   [Container Builder build history](https://console.cloud.google.com/gcr/builds?project=gcp-variant-transforms-test)
+   and verifly that the latest integraion tests have passed.
+1. Choose the appropriate version number by following the semantics in
+   https://semver.org/. See the **Summary** section for a quick overview.
+1. Update the version number in
+   [setup.py](https://github.com/googlegenomics/gcp-variant-transforms/blob/master/setup.py)
+   by submitting a change. Please note the commit SHA generated by GitHub (it's
+   a long string like `4ff1fdfb3957a115a80fe9343306952cd98b6e5c`). You may run
+   the following command to make the next steps easier:
+   `export COMMIT_SHA=<commit sha>`.
+1. Wait until the
+   [Cloud Build](https://console.cloud.google.com/gcr/builds?project=gcp-variant-transforms-test)
+   process builds a new image that contains the change submitted in the previous
+   step. The image has the form
+   `gcr.io/gcp-variant-transforms-test/gcp-variant-transforms:${COMMIT_SHA}`.
+1. The tests in the previous step only cover presubmit tests. We require the
+   release image to pass all tests including `huge_tests`. Run the following on
```

Comment:
I think you can automate this step by creating a new build trigger that is sensitive to tags (instead of a branch) and uses a cloudbuild config that includes `huge_tests` too. So basically after the setup.py change, we create a release tag on the git repo which triggers the above new build trigger (and minimizes possibility of mistakes in manual steps like the one below). For now please add a TODO to replace this step.

Reply:
That's an interesting suggestion! I had not considered the build trigger on tags :) I have actually been thinking that we can automated most of these steps by creating a new `release.sh` script. For now, I added a TODO, but we can do it later. Filed #170 for this.